[Bug] Weird color issue

# Describe the bug.

Tried to build the poster and got this error:

```

Overfull \hbox (5.63pt too wide) in paragraph at lines 344--344
[][]

./example.tex:344: Package color Error: Argument `-0.11766' not in range [0,1].


See the color package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.344 }

./example.tex:344:  ==> Fatal error occurred, no output PDF file produced!
Transcript written on example.log.
make: *** [all-via-pdf] Error 1
```


# Steps to reproduce the bug.

1. clone the repo
2. `git checkout issue2` to incorporate the bugfix for #2. 
3. type make
4. fail to build a pdf.


## Are there additional replication details?

 - Operating System: MacOSX
 
# What is the expected behavior?

The pdf uploaded to the repo should have been generated. It wasnt. 

# How can this issue be closed?

This issue can be closed with a fix that clears this error.

--- a/example.tex
+++ b/example.tex
@@ -147,7 +147,7 @@ PI: {\textbf{Prof. Katy Huff}}, \href{mailto:kdhuff@illinois.edu}{\underline{kdh
 \begin{minipage}{.2\linewidth}
     \includegraphics[width=\linewidth]{img/cyclus.png}\\
     \vspace{20pt}
-    \textcolor{altgeldorange!150}{\textbf{\fontsizemoltres{\texttt{Moltres}}}}\\\vspace{-15pt}
+    \textcolor{altgeldorange!100}{\textbf{\fontsizemoltres{\texttt{Moltres}}}}\\\vspace{-15pt}
 \end{minipage}
 }
 };

